src/screens/Auth/AgeBackScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from 'react'
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native'
+
+function AgeBackScreen(): ReactElement {
```

Comment:
@gohabereg @Bukilan вы сошлись на использовании стрелочных функций, или обычных функций?
```suggestion
const AgeBackScreen: React.FC = () => {
```